src/SimpleWallet/SimpleWallet.cpp

```diff
@@ -561,6 +752,48 @@ void printListTransfersItem(LoggerRef& logger, const WalletLegacyTransaction& tx
   logger(INFO, rowColor) << " "; //just to make logger print one endline
 }
 
+void printListMessagesItem(
```

Comment:
Inside this function each tab character must be replaced with 4 spaces. Please, never use tabs for indentation!